scripts/autoplacement_96_well.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
+from EPPs.common import Autoplacement
+
+
+# Script for performing autoplacement of samples from multiple input plates into a single output plate.
+# If 1 input and 1 output then 1:1 placement. If multiple input plates then
+# takes all samples from each plate before next plate Loops through all inputs and assigns them to the next available space
+# by column-row in the output plate
+class Autoplacement96well(Autoplacement):
```

Comment:
Same comment as above, the comment should be a docstring

Reply:
updated